src/shared/clients/ecsClient.ts

```diff
@@ -59,10 +59,20 @@ export class DefaultEcsClient {
         return containerNames ?? []
     }
 
-    public async listTasks(cluster: string, serviceName: string, taskStatus?: string): Promise<string[]> {
+    public async listTasks(
+        cluster: string,
+        serviceName: string,
+        taskStatus?: string,
+        maxResults?: number
+    ): Promise<string[]> {
```

Comment:
Is there a reason the parameters cannot be a single `args: ECS.ListTasksRequest` parameter? Can use `Omit` if you want to prevent some properties for some reason.